latest_builds/matrix.yml

```diff
@@ -50,20 +65,22 @@ testing-ansible-ansible-10:
       version_specifier: "~=10.0a"
       dists:
         - jammy
-testing-ansible-ansible-11:
-  github_branch: ansible-11
+testing-ansible-ansible-12:
+  github_branch: ansible-12
```

Comment:
Why do you replace `testing-ansible-ansible-11` by `testing-ansible-ansible-12` (while `testing-ansible-ansible-10` is still there)?

Reply:
`jammy` doesn't support anything newer than `ansible 10`, whereas `noble` and newer support `ansible 12`, `testing-ansible-ansible-XX` is declaring which version of ansible you'll get when using the [`ansible-testing`](https://launchpad.net/~ansible/+archive/ubuntu/testing-ansible) PPA